### PR TITLE
Added support to search for reports of moons only.

### PR DESCRIPTION
**Pull request by kejith**
In preparation to get some moon specific reports from the API
- Search Field added
- Postgres Query updated

WfPKf2t
- Added OnClick-Event to the default galaxy sync button to upload system data to the api

### CODE REVIEW

File: public/setup.user.js [hunk 1]
```diff
@@ -11,7 +11,7 @@
 // @grant        GM_xmlhttpRequest
 // @grant        unsafeWindow
 // @connect      localhost
-// @connect      uni2.teamview.mietemieze.de
```
Comment (anoff) [inline public/setup.user.js:14]: um... well... no :D 
guess we have to find a better way to configure this now that multiple people use it 👀

File: userscript/features/galaxyview.js [hunk 2]
```diff
@@ -311,7 +311,7 @@ function doUploadPlanets () {
   }
 }
 
-function addUploadSection () {
+function addUploadSection() {
```
Comment (anoff) [inline userscript/features/galaxyview.js:314]: please set your code formatter to `js standard` :)

**Review by anoff — CHANGES_REQUESTED** [review]
Thanks for the PR. Good idea to add that property.

But please keep the PR to the functional change not reformatting the files

File: userscript/features/galaxyview.js (hunk 2 above)
Comment (kejith) [reply userscript/features/galaxyview.js:314]: `please set your code formatter to js standard :)`
Will do

File: public/setup.user.js (hunk 1 above)
Comment (kejith) [reply public/setup.user.js:14]: We could build this file too, with a Setting to the API in the .env file? Could make it happen.

**Review by kejith — COMMENTED** [review]
So this should work right?

File: userscript/ui/tabSearchReports.html [hunk 3]
```diff
@@ -44,7 +44,14 @@
               class="location-field"> -
             <input type="number" id="system_max" value="" min="1" max="400" class="location-field">
           </td>
-          <td></td>
+          <td>
+            <label for="is_moon">Mond: </label>
```
Comment (anoff) [inline userscript/ui/tabSearchReports.html:48]:
```suggestion
            <label for="is_moon">Moon: </label>
```

for now english. If you know how to add language support, let's talk :D

**Review by anoff — APPROVED** [review]
LGTM

Sorry for the delay 🌝

**Pull request merged**